tests/runner/src/env.rs

```diff
@@ -0,0 +1,166 @@
+// Copyright 2022 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::fmt::Display;
+use std::time::Duration;
+
+use async_trait::async_trait;
+use client::api::v1::codec::SelectResult;
+use client::api::v1::column::SemanticType;
+use client::api::v1::ColumnDataType;
+use client::{Client, Database as DB, Error as ClientError, ObjectResult, Select};
+use comfy_table::{Cell, Table};
+use sqlness::{Database, Environment};
+use tokio::process::{Child, Command};
+use tokio::time;
+
+use crate::util;
+
+pub struct Env {}
+
+#[async_trait]
+impl Environment for Env {
+    type DB = GreptimeDB;
+
+    async fn start(&self, mode: &str, _config: Option<String>) -> Self::DB {
+        match mode {
+            "local" => Self::start_local().await,
+            "remote" => Self::start_remote().await,
+            _ => panic!("Unexpected mode: {}", mode),
+        }
+    }
+
+    /// Stop one [`Database`].
+    async fn stop(&self, _mode: &str, mut database: Self::DB) {
+        database.server_process.kill().await.unwrap()
+    }
+}
+
+impl Env {
+    pub async fn start_local() -> GreptimeDB {
+        let server_process = Command::new("cargo")
+            .current_dir("../")
+            .args(["run", "--", "datanode", "start"])
```

Comment:
is "datanode" should be changed into "standalone"?

Reply:
Good catch!